redbox-core/redbox/models/chain.py

```diff
@@ -128,14 +128,22 @@ class AISettings(BaseModel):
     chat_map_question_prompt: str = CHAT_MAP_QUESTION_PROMPT
     reduce_system_prompt: str = REDUCE_SYSTEM_PROMPT
     reduce_question_prompt: str = REDUCE_QUESTION_PROMPT
-    llm_max_tokens: int = 1024
 
-    # size: int = 19 rag_k
     match_boost: int = 1
-    # num_candidates: int = 13 rag_num_candidates
     knn_boost: int = 1
     similarity_threshold: int = 0
 
+    chat_backend: Literal["azure", "openai", "fake"] = "azure"
+    fake_backend_responses: list = ["hello"]
```

Comment:
I hate me too... but I cant think of another way. all ideas welcome

Reply:
Could we have the fake llm expect an env var and read it's responses from there? In tests we could set that var for the test/session as needed